.Rprofile

```diff
@@ -3,3 +3,11 @@ options(
   blogdown.rmd = TRUE,
   blogdown.subdir = "articles"
 )
+
+if (interactive()) {
```

Comment:
Shouldn't you always source the user profile?

Reply:
Yeah you are probably right